Source/WebCore/css/CSSPrimitiveValueMappings.h

```diff
@@ -107,12 +186,11 @@ DEFINE_TO_FROM_CSS_VALUE_ID_FUNCTIONS
 #undef TYPE
 #undef FOR_EACH
 
-template<> inline CSSPrimitiveValue::operator ColumnSpan() const
+template<> inline ImplicitValueMapper::operator ColumnSpan() const
 {
-    // Map 1 to none for compatibility reasons.
-    if ((primitiveUnitType() == CSSUnitType::CSS_NUMBER || primitiveUnitType() == CSSUnitType::CSS_INTEGER) && m_value.number == 1)
+    if (downcast<CSSPrimitiveValue>(m_value).doubleValue() == 1)
         return ColumnSpan::None;
-    return fromCSSValueID<ColumnSpan>(valueID());
+    return fromCSSValueID<ColumnSpan>(downcast<CSSPrimitiveValue>(m_value).valueID());
```

Comment:
Can we store the result of the downcast in a variable?

Reply:
I’d prefer not to, but I suppose it really doesn’t matter since this code will be deleted soon enough.